echion/threads.h

```diff
@@ -28,6 +28,76 @@
 #include <echion/tasks.h>
 #include <echion/timing.h>
 
+
+#if defined PL_LINUX
+#define MAX_FD_COUNT 16
```

Comment:
Is there a reasoning behind setting this to 16?

Reply:
The default in many cases should be 1024. 16 felt like a good compromise. No other reason beyond this.

Reply:
I'm ok with having 16, but prefer it to be configurable without recompilation. Imagine a customer seeing cpu spikes with many threads due to `open/proc/stat` calls. Then, we'd need to do a new release for them to try different `MAX_FD_COUNT`. It would be easier for us to ask them to try a different environment variable for this.